packaging/debian/storagenode.config

```diff
@@ -119,7 +121,7 @@ case "$STATE" in
             if db_go; then
                   db_get storagenode/wallet
                   wallet="$RET"
-                  if [[ ! "${wallet}" =~ ^0x*$ ]] ; then
+                  if ! [[ "${wallet}" =~ ^0x.*$ ]] ; then
```

Comment:
This can be refined even more to ^0x.{40}$

Reply:
It looks like this is just checking for an invalid prefix, while the number of chars is checked on line 126

Reply:
Indeed, it's to have different messages for the two types of errors.